### PR TITLE
(maint) update github io plugin list.

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: assets/changes/catalog/parsed_plugin_list.js [hunk 1]
```diff
@@ -229,12 +289,24 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-collector-users"
   },
   {
+    "name": "Anomalydetection",
+    "type": "Processor",
+    "description": "snap plugin intended to process data and highlight outliers",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 2]
```diff
@@ -229,12 +289,24 @@ myfcn(
...
+    "url": "https://github.com/intelsdi-x/snap-plugin-processor-anomalydetection"
+  },
+  {
     "name": "Movingaverage",
     "type": "Processor",
     "description": "Process snap metrics and return a moving average",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
+  {
+    "name": "Couchbase",
+    "type": "Collector",
+    "description": "No description available.",
+    "url": "https://github.com/Staples-Inc/snap-plugin-collector-couchbase"
+  },
+  {
+    "name": "Netstat",
+    "type": "Collector",
+    "description": "Collect TCP connection metrics for the snap telemetry framework",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 4]
```diff
@@ -1,6 +1,60 @@
...
+  {
+    "name": "Nginx",
+    "type": "Collector",
+    "description": "Collect nginx metrics from the status endpoint",
+    "url": "https://github.com/Staples-Inc/snap-plugin-collector-nginx"
+  },
+  {
+    "name": "Procstat",
+    "type": "Collector",
+    "description": "Collect process metrics using the snap telemetry framework",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 5]
```diff
@@ -1,6 +1,60 @@
...
+    "url": "https://github.com/Staples-Inc/snap-plugin-collector-procstat"
+  },
+  {
+    "name": "Blueflood",
+    "type": "Publisher",
+    "description": "A snap blueflood publisher",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 6]
```diff
@@ -229,12 +289,24 @@ myfcn(
...
   {
+    "name": "Processor",
+    "type": "Statistics",
+    "description": "Process metrics and return statistics over sliding window.",
+    "url": "https://github.com/intelsdi-x/snap-plugin-processor-statistics"
+  },
+  {
     "name": "Tag",
     "type": "Processor",
     "description": "Process snap metrics and add tags to label the data",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: **S**nap*

File: assets/changes/catalog/parsed_plugin_list.js [hunk 7]
```diff
@@ -1,6 +1,60 @@
 myfcn(
 [
   {
+    "name": "Ping",
+    "type": "Collector",
+    "description": "",
```
Comment (kindermoumoute) [inline assets/changes/catalog/parsed_plugin_list.js]: No description available?

**Comment by nanliu** [comment]
@kindermoumoute this is auto generated, I can use a regex to capitalize Snap. I can't update the raintank plugin description, since I don't have admin access to their repo. I can make a default setting of 'no description', would that work?

**Comment by kindermoumoute** [comment]
Ok, I guess it's repo maintainer responsibility to change it. I did the changes on repo I have right on.
LGTM.

**Comment by sarahjhh** [comment]
LGTM 
@kindermoumoute- fyi I requested this update from Nan in order to have the github.io page plugin catalog updated.

**Pull request merged**